helper/inet/inet.go

```diff
@@ -0,0 +1,81 @@
+package inet
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+)
+
+const (
+	advertiseIp = iota
+	advertisePort
+)
+
+const (
+	address = iota
+	cidr
+)
```

Comment:
These constants make it much harder to read the code that uses them. We should just use the explicit index in the slices directly for clarity.